src/amp.js

```diff
@@ -49,6 +50,7 @@ try {
       installHistoryService(window);
       viewerFor(window);
       vsyncFor(window);
+      templatesFor(window);
```

Comment:
They were always compiled in because runtime.js registers the `AMP.registerTemplate` function. The issue was that depending on download order, e.g. whether core's templates service would be installed (if template implementation is downloads first and calls `AMP.registerTemplate`) or another version, downloaded with `amp-list` or similar using element, would be installed first. I didn't like this ambiguity. This is subject to the service/implementation refactoring.